include/elliptics/packet.h

```diff
@@ -747,7 +747,9 @@ struct dnet_io_attr
 	 */
 	uint64_t		total_size;
 
-	uint64_t		reserved1;
+	/* Combination of DNET_RECORD_FLAGS_* */
+	uint64_t		record_flags;
```

Comment:
How does it differ from io->flags? There will be a serious intersection between the two - we should implement it the way that does not introduce duplication.